requirements.txt

```diff
@@ -10,4 +10,4 @@ stopit >= 1.1.1, < 1.1.999
 yamlconf >= 0.2.2, < 0.2.999
 pytest >= 3.2.2, < 3.2.999
 pyyaml == 3.11
-revscoring >= 2.0.2, < 2.0.999
+revscoring >= 2.0.2, < 2.1.999
```

Comment:
You don't want a minimum version of 2.1.x?

Reply:
Minimum version of 2.1.x?  what do you mean?

Reply:
I think awight means we drop support of 2.0.x

Reply:
Ahh.  I don't see that as necessary in order to build models that are compatible, but I'm OK with it if y'all feel strongly.

Reply:
I don't feel anything right now, The coffee hasn't kicked in yet.